src/status_im/contexts/onboarding/intro/view.cljs

```diff
@@ -11,67 +11,91 @@
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
+(defn- terms
+  [terms-accepted? set-terms-accepted?]
```

Comment:
(I know you didn't write the function and minor neatpick)
the question mark at the end of `set-terms-accepted?` threw me a bit off, I expected it to be either a function returning a boolean, or a boolean, but then you see that is an updater, `set-terms-accepted!` seems more reasonable to me since it's a side effect, took me a while to realize that its' the usestate updater

Reply:
I get now the reasoning, `set` + `accepted-terms?`, we are setting the `accepted-terms?` variable, I am probably a bit slow, but I found that rather confusing

Reply:
This is an interesting perspective. I agree with you this can be confusing and it only works well if everybody understands the convention `set-*`, where `*` is always identical to the read function. I wouldn't mind either way, but it's better if it's brought up to more devs to see what they think/prefer.